src/did_resolve.rs

```diff
@@ -676,9 +675,11 @@ impl DIDResolver for HTTPDIDResolver {
             }
         };
         let https = HttpsConnector::new();
+
```

Comment:
```suggestion
```

Reply:
Also about https://github.com/spruceid/ssi/pull/209/files#diff-84938e3eaf160d50bb356f2f7428ca3511ed765cd5f046ee066f52095056ab4bL15:

A common convention I think is to not add/remove whitespace unless already changing that code, or in a separate commit that just changes whitespace/formatting. (This might also be considered nitpicking...)